Makefile

```diff
@@ -1,11 +1,17 @@
 VERSION=$(shell git tag|tail -n1)
 R2W=r2wars-$(VERSION)
 
+ifeq ($(OS),Windows_NT)
+    build := msbuild
```

Comment:
Do not indent here

Reply:
Strange... msbuild works in mac too. Which version of mono are u using? Because im using mono on mac with msbuild

Reply:
I installed mono 5.14 from the packages provided by mono-project.com
Their packages (at least for linux, the others I don't know) do not contain msbuild.
According to https://lists.dot.net/pipermail/mono-list/2017-May/052476.html, msbuild is now a separate component and could be compiled from source from their github page.